E3db/Classes/Application.swift

```diff
@@ -16,15 +16,42 @@ public class Application {
     public init(apiUrl: String, appName: String, realmName: String, brokerTargetUrl: String) {
         self.apiUrl = apiUrl
         self.appName = appName
-        self.realmName = realmName
+        self.realmName = realmName.lowercased()
```

Comment:
i updated the server code to handle the conversion of realm name to domain name more intelligently, so i think this lowercasing should no longer be necessary in the most recent build of identity service.